scripts/templates/ldrddi.cpp.mako

```diff
@@ -443,15 +443,17 @@ ${tbl['export']['name']}(
         {
             // return pointers to loader's DDIs
             %for obj in tbl['functions']:
+            if (version >= ${th.get_version(obj)}) {
```

Comment:
Since this creates one if() per API, it works fine...
We should find a way in later edition to have on if() per version and place API's in proper if
functionally, it won't make a difference, so it can go in today as is, but we can clean this up later.

Reply:
That may require nested loops, or make assumptions that the spec itself always passes APIs in order of version... Either way seems like a more complicated design that will take more time that we have at the moment.